src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -0,0 +1,146 @@
+ *
+ * The fieldsComponent is passed some state props: inReview (always true here) and editing.
+ */
+
+export default class ReviewCollapsibleChapter extends React.Component {
+  constructor() {
+    super();
+    this.handleSave = this.handleSave.bind(this);
+    this.handleEdit = this.handleEdit.bind(this);
+    this.scrollToTop = this.scrollToTop.bind(this);
+    this.toggleChapter = this.toggleChapter.bind(this);
+    this.state = { open: false };
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  focusOnPage(key) {
+    const pageDiv = document.querySelector(`#${key}`);
+    focusElement(pageDiv);
+  }
+
+  handleEdit(key, editing) {
+    this.props.onEdit(key, editing);
+    this.scrollToPage(key);
+    this.focusOnPage(key);
+  }
+
+  handleSave(key) {
+    this.props.onEdit(key, false);
+    this.scrollToPage(key);
+    this.focusOnPage(key);
+  }
+
+  scrollToTop() {
+    scroller.scrollTo(`chapter${this.props.chapterKey}ScrollElement`, {
+      duration: 500,
+      delay: 2,
+      smooth: true,
+    });
+  }
+
+  scrollToPage(key) {
+    scroller.scrollTo(`${key}ScrollElement`, {
+      duration: 500,
+      delay: 2,
+      smooth: true,
+    });
+  }
+
+  toggleChapter() {
+    const isOpening = !this.state.open;
+    this.setState({ open: !this.state.open }, () => {
+      if (isOpening) {
+        this.scrollToTop();
+      }
+    });
+  }
+
+  render() {
+    let pageContent = null;
+    if (this.state.open) {
+      pageContent = (
+        <div id={`collapsible-${this.id}`} className="usa-accordion-content">
+          {this.props.pages.map(page => {
+            const editing = this.props.data[page.pageKey].editMode;
+
+            return (
+              <div key={page.pageKey} className="form-review-panel-page">
+                <Element name={`${page.pageKey}ScrollElement`}/>
+                <FormPage
+                    reviewPage
+                    hideTitle={this.props.pages.length === 1}
+                    onEdit={() => this.handleEdit(page.pageKey, !editing)}
+                    onSubmit={() => this.handleSave(page.pageKey)}
+                    reviewMode={!editing}
+                    route={{ pageConfig: page }}/>
+              </div>
+            );
+          })}
+        </div>
+      );
+    }
+
+    return (
+      <div id={`${this.id}-collapsiblePanel`} className="usa-accordion-bordered form-review-panel">
+        <Element name={`chapter${this.props.chapterKey}ScrollElement`}/>
+        <ul className="usa-unstyled-list">
+          <li>
+            <div className="accordion-header clearfix" >
+              <button
+                  className="usa-button-unstyled"
+                  aria-expanded={this.state.open ? 'true' : 'false'}
+                  aria-controls={`collapsible-${this.id}`}
+                  onClick={this.toggleChapter}>
+                {this.props.chapter.title}
+              </button>
+            </div>
+            {this.state.open && pageContent}
```

Comment:
Do you need to have this check for `this.state.open` here if you have it above before setting the value of `pageContent`?